src/main/java/picard/sam/CreateSequenceDictionary.java

```diff
@@ -245,25 +257,32 @@ protected int doWork() {
                 }
             }
         } catch (IOException e) {
-            throw new PicardException("Can't write to or close output file " + OUTPUT.getAbsolutePath(), e);
+            throw new PicardException("Can't write to or close output file " + OUTPUT.getURIString(), e);
         } catch (IllegalArgumentException e) {
             // in case of an unexpected error delete the file so that there isn't a
             // truncated result which might be valid yet wrong.
-            OUTPUT.delete();
+            OUTPUT.toPath().toFile().delete();
```

Comment:
Hm - does `OUTPUT.toPath().toFile()` return a legitimate, useable File for a remote path ?